solutions/devsprint-ana-nogal-1/DeliveryApp/Screens/RestaurantList/RestaurantCell.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  RestaurantCell.swift
+//  DeliveryApp
+//
+//  Created by Cora on 27/04/22.
+//
+
+import UIKit
+
+class RestaurantCell: UITableViewCell {
+    
+    static let identifier = "restaurantCell"
+
+    lazy var image: UIImageView = {
+        let img = UIImageView()
+        img.image = UIImage(imageLiteralResourceName: "restaurant-logo")
+        img.translatesAutoresizingMaskIntoConstraints = false
+        return img
+    }()
+    
+    lazy var title: UILabel = {
+        let label = UILabel()
+        label.text = "Benjamin a Padaria"
```

Comment:
O texto esta hardcoded... o melhor será inicializar a vazio.
```suggestion
        label.text = ""
```